airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1301,8 +1302,8 @@ class BigQueryUpdateDatasetOperator(BaseOperator):
 
     @apply_defaults
     def __init__(self,
-                 dataset_id: str,
                  dataset_resource: dict,
+                 dataset_id: Optional[str] = None,
```

Comment:
This change in order of arguments probably needs a note in UPDATING.md as well

Reply:
Added